azure-stack/operator/azure-stack-download-azure-marketplace-item.md

```diff
@@ -110,100 +112,68 @@ Once you have registered, you can disregard the following message that appears o
+5. To select the Marketplace items such as VM images, extensions, or solution templates to download, run the following command. 
 
    ```powershell
-   Export-AzSOfflineResourceProvider -destination "Destination folder path" -azCopyDownloadThreads "AzCopy threads number" -azureContext $AzureContext
+   $products = Select-AzsMarketplaceItem
    ```
+  * This will first pop up a Powershell Grid listing all the Azure Stack Registrations available in the selected subscription. Pick the registration that matches the Azure Stack environment you're downloading the marketplace items for and click OK.
 
-   The `-azCopyDownloadThreads` parameter is optional. It should only be used when you have a low-bandwidth network, and you are using premium download. This option specifies the number of concurrent operations in AzCopy. If you are running a low-bandwidth network, you can specify a lower number to avoid failure caused by resource competition. You can see more details in [this Azure article](/previous-versions/azure/storage/storage-use-azcopy#specify-the-number-of-concurrent-operations-to-start).
-
-   The `-azureContext` parameter is also optional. If you do not specify the Azure context, then the cmdlet will use the default Azure context.
-
-7. When the tool runs, you should see a screen similar to the following image, with the list of available Azure Marketplace items:
-
-   ![Marketplace items](media/azure-stack-download-azure-marketplace-item/tool1.png)
-
-8. If more than one version of a Marketplace item is available, the **Version** column shows **Multiple versions**. If the version of an item is shown as **Multiple versions**, you can select that item and then choose a specific version from the resulting version selector window.
-
-9. Select the item that you want to download and make a note of the **Version**. You can hold the **Ctrl** key to select multiple images. You reference the *version* when you import the item in the next procedure.
+     ![Select Azure Stack Registrations](media/azure-stack-download-azure-marketplace-item/selectRegistration.png)
 
+  * You should now see a second PowerShell Grid listing all the marketplace items available for download. Select the item that you want to download and make a note of the **Version**. You can hold the **Ctrl** key to select multiple images.
+     ![Select Azure Stack Registrations](media/azure-stack-download-azure-marketplace-item/selectProducts.png)
+  
     You can also filter the list of images by using the **Add criteria** option.
+     ![Select Azure Stack Registrations](media/azure-stack-download-azure-marketplace-item/selectProductswithFilter.png)
 
-10. If you have not installed the Azure Storage tools, you will get the following message. In order to install these tools, make sure you download [AzCopy](/azure/storage/common/storage-use-azcopy#download-azcopy):
+    Once you've made your selections. Click OK.
 
```

Comment:
Put a comma after "selections". Should be "Once you've made your selections, select OK."